docs/app/Examples/views/Comment/States/CommentExampleCollapsed.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react'
+import { Checkbox, Comment } from 'semantic-ui-react'
+
+export default class CommentExampleCollapsed extends Component {
```

Comment:
This is a cool example, really like the simple interactivity to show how a prop works.